src/components/dynamic/admin/Dashboard.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import Title from "./Title.jsx";
+import Filters from "./Filters.jsx";
+import Toolbar from "./dashboards/Toolbar.jsx";
+import Table from "./Table.jsx";
+import {
+  FILTERS,
+  HEADERS,
+  STATUSES,
+  TAGS,
+} from "@/data/dynamic/admin/Admins.js";
+
+const Dashboard = () => {
+  const [admin, setAdmin] = useState(null);
```

Comment:
these should be in the parent comp and simply just passed down into the title/toolbar/filters/table comps